app/services/magic_link_flow.rb

```diff
@@ -0,0 +1,26 @@
+class MagicLinkFlow
+  def self.sign_up(candidate)
```

Comment:
The name of this method implies some flexibility, or to put it another way, some mixed respinsibility, to `MagicLinkFlow`. Could we maybe push this language up into the class name — something like `MagicLinkSignupFlow#begin`?